File: test/fixtures/python/redirect_variable_data.py
import os
import requests

headers = {
    'Content-Type': 'application/x-www-form-urlencoded',
}

data = 'foo'
with open(os.getenv('FILENAME')) as f:
    data += '&' + f.read().replace('\n', '').replace('\r', '')

response = requests.post('http://localhost:28139', headers=headers, data=data.encode())

File: test/fixtures/node/redirect_variable_data.js
import fetch from 'node-fetch';
import * as fs from 'fs';

fetch('http://localhost:28139', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/x-www-form-urlencoded'
  },
  body: new ArrayBuffer('foo&', fs.readFileSync(process.env['FILENAME'], 'utf-8'))
});
